data/tmc-devices.json

```diff
@@ -195,19 +194,19 @@
                         },
                         "properties": {
                             "CspSubarrayLNFQDN": [
-                                "ska_mid/tm_leaf_node/csp_subarray01"
+                                "ska_mid/tm_leaf_node/csp_subarray02"
                             ],
                             "DishLeafNodePrefix": [
                                 "ska_mid/tm_leaf_node/d"
                             ],
                             "SdpSubarrayLNFQDN": [
-                                "ska_mid/tm_leaf_node/sdp_subarray01"
+                                "ska_mid/tm_leaf_node/sdp_subarray02"
                             ],
                             "CspSubarrayFQDN": [
-                                "mid_csp/elt/subarray01"
+                                "mid_csp/elt/subarray_02"
                             ],
                             "SdpSubarrayFQDN": [
-                                "mid_sdp/elt/subarray_1"
+                                "mid_sdp/elt/subarray_2"
                             ],
                             "SkaLevel": [
```

Comment:
SkaLevel should be 2 for SubarrayNode. Also, check the SkaLevel for other devices.

Reply:
'skalevel' updated for all the devices.